vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -362,6 +365,19 @@ func (d *VolumeDriver) MountVolume(name string, fstype string, id string, isRead
 
 // private function that does the job of mounting volume in conjunction with refcounting
 func (d *VolumeDriver) processMount(r volume.MountRequest) volume.Response {
+	// get volume metadata
+	volumeMeta, err := d.GetVolume(r.Name)
```

Comment:
If `plugin_utils.AlreadyMounted(r.Name, d.mountRoot)` then you can skip `d.GetVolume(r.Name)` right?